receiver/apachereceiver/metadata.yaml

```diff
@@ -76,6 +76,129 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [ server_name ]
+  apache.bytes_per_request:
+    enabled: true
+    description: The average number of bytes per request.
+    unit: By
+    sum:
+      value_type: double
+      input_type: string
+      monotonic: false
+      aggregation: delta
+    attributes: [ server_name ]
+  apache.bytes_per_second:
+    enabled: true
+    description: The average number of bytes per second.
+    unit: By
+    sum:
+      value_type: double
+      input_type: string
+      monotonic: false
+      aggregation: delta
+    attributes: [ server_name ]
+  apache.cpu_children_system:
+    enabled: true
+    description: Jiffies used in system mode by children processes.
+    unit: s
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: [ server_name ]
+  apache.cpu_children_user:
+    enabled: true
+    description: Jiffies used in user mode by children processes.
+    unit: s
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: [ server_name ]
+  apache.cpu_system:
+    enabled: true
+    description: Jiffies used in system mode.
+    unit: s
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: [ server_name ]
+  apache.cpu_user:
+    enabled: true
+    description: Jiffies used in user mode.
+    unit: s
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: [ server_name ]
```

Comment:
The OTel data model strongly prefers that such metrics be combined, and that the data points are differentiated based on attributes. 

In this case, we could have one or two metrics, depend on whether we decide self + children is a meaningful total. (I think probably it is, but what do you think?)

```
  # 1 metric
  apache.cpu.time:
    description: Jiffies used by category.
    ...
    attributes: [ server_name, level (self | children), mode (system | user) ]

  # 2 metrics
  apache.cpu.time.self:
    description: Jiffies used by category.
    ...
    attributes: [ server_name, mode (system | user) ] 
  apache.cpu.time.children:
    description: Jiffies used by category.
    ...
    attributes: [ server_name, mode (system | user) ] 
```

Reply:
I do agree, it makes sense.